MediaBrowser.Providers/MediaInfo/ProbeProvider.cs

```diff
@@ -141,34 +142,30 @@ public bool HasChanged(BaseItem item, IDirectoryService directoryService)
                 && item.SupportsLocalMetadata
                 && !video.IsPlaceHolder)
             {
-                if (!video.SubtitleFiles.Order().SequenceEqual(
-                        _subtitleResolver.GetExternalFiles(video, directoryService, false)
-                            .Select(info => info.Path).Distinct().Order().ToList(),
-                        StringComparer.Ordinal))
+                var externalFiles = new HashSet<string>(_subtitleResolver.GetExternalFiles(video, directoryService, false).Select(info => info.Path), StringComparer.OrdinalIgnoreCase);
```

Comment:
Should we be using ignore case for file paths?

Reply:
Actually a good point. Depending on OS this might not be correct. What's your preference?